SRM: Release Notes for 1.1.3 (#3457)
harness/developer-hub: release-notes/service-reliability-management.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

## Latest: Version 1.1.3

### New features and enhancements

> SRM Dashboards now include comprehensive data for Composite Service Level Objectives (Composite SLOs). This enhancement offers users a holistic view of both Simple and Composite SLO performance. (SRM-15419)

Reviewer: Is SRM Dashboards (with the uppercase D) the name of the dashboard component? If yes, we'll need to write "includes". If it's not a proper noun, we probably don't require the uppercase D. One or the other.

Also, should "Composite Service Level Objectives" be in the uppercase? How about "Simple"?

```suggestion
SRM Dashboards now include comprehensive data for Composite Service Level Objectives (Composite SLOs). This enhancement gives you a holistic view of both Simple and Composite SLO performance. (SRM-15419)
```

### Early access features
This release does not include any early access features.

### Fixed issues

> - Firehydrant notifications were not being triggered as expected for monitored services without any configured Service Level Objectives (SLOs). (SRM-15458)

Reviewer: ```suggestion
- FireHydrant notifications were not being triggered as expected for monitored services without any configured Service Level Objectives (SLOs). (SRM-15458)
```

>   This issue has been successfully resolved. Firehydrant notifications will now be triggered for all monitored services, irrespective of whether SLOs are configured.

Reviewer: ```suggestion
  This issue has been resolved. FireHydrant notifications will now be triggered for all monitored services, irrespective of whether SLOs are configured.
```
